system/include/libcxx/__config

```diff
@@ -308,7 +308,7 @@
    // random data even when using sandboxing mechanisms such as chroots,
    // Capsicum, etc.
 #  define _LIBCPP_USING_ARC4_RANDOM
-#elif defined(__Fuchsia__) || defined(__wasi__)
+#elif defined(__Fuchsia__) || defined(__wasi__) || defined(__EMSCRIPTEN__)
```

Comment:
Mention in system/lib/libcxx/readme.txt?

Reply:
Good idea, added. Also I added a mention in the changelog.